cherryontech/do-five-things
# feat: Settings inputs

## Context
A user trying to enter their tasks

## Work done
- Created task input component for use on the settings page
- Created a UID helper so that each input will be unique (this is esp important for SR users, as it ensures they'll hear the correct label with the correct input, no matter how many similar inputs there are)
- Put 5 of the inputs on the settings page, and show the response (for demo).

## Manual testing instructions
1. serve app
2. Navigate to /settings
3. Type in the inputs
- Expect to see the output from each of the inputs listed below the group of inputs
4. Open the Accessibility panel of Dev Tools and select an input
- Expect to see its label as "Task <number>."
